### ISSUE TITLE
[Enhancement] Add link to youtube video on the video error page

### ISSUE THREAD

**Issue by Half-and-Xat** [iv-org/invidious#4632]
EDIT by @unixfox 

This enhancement is for adding the link to the original youtube.com video on the error page. Like it's already possible when the instance is blocked by youtube.

----------

The current issue #4584 leads to an error page. This behavior results in a loose of navigation features. The absolute minimum I would expect is a link to the YouTube video page (including embed link). This way Invidious is still useful for searching videos when videos themselves cannot be loaded. It's a must-have improvement in my opinion, especially since it's not the first time that Google break things.

Beyond the minimum: every navigation like link to profile, video successions etc improves the usability even further. I don't know if video downloads are affected by the bug, but if not, those could also be added. These things are more a nice-to-have.

**Comment by unixfox** [comment]
What do you actually mean by navigation?

When invidious is blocked there is no way to know what is the profile of the video, nor the description and not even the comments.

**Comment by gbschenkel** [comment]
But Invidious have the link which was blocked, is not only take the instance/watch?v=p2Lk3EWwV-I and change it to https://youtube.com/watch?v=p2Lk3EWwV-I ? (used random link video)

**Comment by SamantazFox** [comment]
@syeopite only the most basic error page (triggered by an `InfoException`) has it:

![image](https://github.com/iv-org/invidious/assets/52980486/c94e179b-ed9e-4b49-8931-bb15518ec3e0)

**Comment by Half-and-Xat** [comment]
And what is the argument against doing it better? Does it hurt to have it directly on error page? I don't think so.

Keep in mind that Invidious is not just for people like us, people who know how to use URLs correctly, people who are able to trial and error to find the embed version of YouTube (like ``youtube.com/e/p2Lk3EWwV-I`` or ``youtube.com/embed/watch?v=p2Lk3EWwV-I`` or ``youtube.com/embed/p2Lk3EWwV-I``). If I send a link to other people who usually don't use Invidious, they may don't even know that they just need to replace ``instance.domain/watch?v=p2Lk3EWwV-I`` with youtube.com/watch?v=p2Lk3EWwV-I`` or to click on "other instances" to get the YouTube link (even I did not realize that there was a link, too).

@SamantazFox: Even here, ``Go to YouTube (Embed)`` (2 links) would be a little improvement. I agree with unifying it.

**Comment by unixfox** [comment]
Link was added in https://github.com/iv-org/invidious/pull/4650/files#diff-3919f4375b028c051402e6e79ae426d16da8fc4db65b4dfa945c384d00132870R150 by switching to InfoException.
